nui/src/main/java/org/terasology/nui/canvas/CanvasControl.java

```diff
@@ -27,14 +27,78 @@
 
     void postRender();
 
+    /**
+     * Call this method when the mouse position changes.
+     * This method does not support multiple pointers. Use {@link #processMousePosition(Vector2i, int)} for that case if supported.
+     * @param position The position of the mouse pointer
+     */
     void processMousePosition(Vector2i position);
 
+    /**
+     * Call this method when the position of a mouse pointer changes.
+     *
+     * The pointers must have a consistent ordering in the positions array e.g.
+     * the first pointer position must be at index 0, the second at index 1, third at index 3 etc.
```

Comment:
I guess it's a typo?
```suggestion
     * the first pointer position must be at index 0, the second at index 1, third at index 2 etc.
```